### DISCUSSION TITLE
Trying to contact sales

### DISCUSSION THREAD

**Discussion by ClundXIII** [CollaboraOnline/online#7496] [General]
Dear Maintainers,

I have been trying to contact sales, so far I have tried:

* using the web form (freezes at "please wait")
* trying to reach out via the Fediverse (no reply)
* trying to message @mmeeks at his gnome address (no longer exists)

Does somebody have an Email Address?

**Comment by vmiklos** [comment]
Next time you want to email Collaborans, please send a mail to hello@collaboraoffice.com.

In this specific case, our team received your message. They will contact you shortly. :-) Thanks!
